platform/config-core/src/main/java/org/motechproject/config/core/utils/ConfigPropertiesUtils.java

```
@@ -115,6 +117,31 @@ public static Properties getPropertiesFromSystemVarString(String string) {
     }
 
     /**
+     * Returns {@code PropertiesConfiguration} and creates config file if does not exist
+     * @param basePath path to the file with config
+     * @param fileName name of the file with config
+     */
+    public static PropertiesConfiguration getPropertiesConfiguration(String basePath, String fileName) {
+        File configFile = new File(basePath, fileName);
+        try {
+            new File(configFile.getParent()).mkdirs();
+            configFile.createNewFile();
+        } catch (IOException e) {
+            throw new MotechConfigurationException(String.format("Cannot create file %s", configFile.getAbsolutePath()), e);
+        }
+
+        PropertiesConfiguration propertiesConfiguration = new PropertiesConfiguration();
+        propertiesConfiguration.setBasePath(basePath);
+        propertiesConfiguration.setFileName(fileName);
+        try {
+            propertiesConfiguration.load();
+        } catch (ConfigurationException e) {
+            LOGGER.error("Unable to load config locations: " + e.getMessage());
```

this should throw an exception, the same as above